CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# UNRELEASED
+
+* BREAKING: Remove Mapit api methods. These are no longer user by any apps, so should not be breaking in practice.
```

Comment:
Minor: think you mean 'used' instead of 'user'.

Reply:
Yes, I do mean used. Ta!